flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -1254,6 +1254,21 @@
                     <WWISE_EVENT>lswitch</WWISE_EVENT>
                 </UseTemplate>
 
+                <!-- Sync strobe+beacon when used simultaneously -->
+                <!-- BEACON1 CIR:15 BUS:2,BEACON2 CIR:16 BUS:3,STROBE1 CIR:23 BUS:3,STROBE2 CIR:24 BUS:3,STROBE3 CIR:25 BUS:3 -->
+                <UseTemplate Name="FBW_Light_Sync">
+                    <SYNC_FREQUENCY>10</SYNC_FREQUENCY>
+                    <SYNC_CONDITION>(A:LIGHT BEACON, bool) (A:LIGHT POTENTIOMETER:24, percent) 0 &gt; and (L:LIGHTING_STROBE_0, number) 2 &lt; and</SYNC_CONDITION>
+                    <SYNC_REQUISITE>(L:A32NX_ELEC_AC_1_BUS_IS_POWERED, Bool) (L:A32NX_ELEC_AC_2_BUS_IS_POWERED, Bool) and</SYNC_REQUISITE>
```

Comment:
**potential issue**: This seems to stop light synchronisation when only one of both buses is powered.
**suggestion**: Change to `or`, to ensure the synchronisation continues.
```suggestion
                    <SYNC_REQUISITE>(L:A32NX_ELEC_AC_1_BUS_IS_POWERED, Bool) (L:A32NX_ELEC_AC_2_BUS_IS_POWERED, Bool) or</SYNC_REQUISITE>
```

Reply:
All strobes sit on the same bus, `Connections:bus.3` = `AC_BUS_2`, so it does not really make any sense to attempt a sync when the strobe is not functioning. Added an extra `els` to this check, so if we lose one BUS, and strobe or beacon dies, we ensure unsynced state, and an sync attempt when BUS is back.